### DISCUSSION TITLE
Сигнализация

### DISCUSSION THREAD

**Discussion by octothorpteam** [octothorp-team/public-dobrograd#598] [Предложения]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/qbaw/32/248_2.png) [qbaw](https://forum.octothorp.team/u/qbaw) ([#9129](https://forum.octothorp.team/p/9129))

Здрасте!
 Короче я хочу предложить идею добавить сигнализацию на машину.

Как это будет работать?
 А легко можно сделать так что если в машину попадет пуля, на неё прыгнет/упадёт человек или проп, но и просто если её взломают то сработает сигнализация которая будет как сигнализация.

Выключить её можно будет 2 способами (время выберите сами, но я думаю что 15-30 секунд будет достаточно)

1. Через какой-то время она сома выключится
2. Просто нажать рукой левой или правой кнопкой мыши по ней (логично это смогут делать только владельцы).

Как её получить?:
 Ну тут либо у механика (за 10-15к думаю сойдёт)
 Либо при покупки машины она уже будет установлена.

По сути это всё внизу выберите надо это вам или нет

UPD: Про слышимость сигнализации я не придумал были идеи что она будет слышна до 2 этажа Линкольн 3 но тут хз
 UPD2: Я думаю что это кто-то да предлагал ну я сам не видел.

- Добавляем
- Не добавляем

0голосов

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/animeshnitsa2009/32/348_2.png) [Animeshnitsa2009](https://forum.octothorp.team/u/Animeshnitsa2009) ([#9260](https://forum.octothorp.team/p/9260))


Уже была такая идея и вроде это в планах

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/qbaw/32/248_2.png) [qbaw](https://forum.octothorp.team/u/qbaw) ([#9276](https://forum.octothorp.team/p/9276))


крута

**Reply by msmr-dev** [reply]
бро забей прост доброград не собираются обновлять, dead moment
